### PR TITLE
upgrade gradle-intellij plugin, address error during snyk-sca

### PR DESCRIPTION
**Pull request by greene858**
During SecRel, snaky-sca was failing 
` 
A problem occurred configuring root project 'abd_vro'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find org.jetbrains.intellij.plugins:structure-base:3.139.
`

Associated tickets or Slack threads:
[- #?](https://dsva.slack.com/archives/C04CA47HV96/p1687449590800199)

## How does this fix it?[^1]
Upgrade gradle.plugin.org.jetbrains.intellij.plugins:gradle-intellij-plugin from 0.4.26 to latest 1.14.1

### CODE REVIEW

**Review by dfitchett — APPROVED** [review]

**Review by chengjie8 — APPROVED** [review]
LGTM

**Review by msnwatson — APPROVED** [review]

File: gradle-plugins/build.gradle [hunk 1]
```diff
@@ -5,6 +5,11 @@
 plugins {
     // Support convention plugins written in Groovy. Convention plugins are build scripts in 'src/main' that automatically become available as plugins in the main build.
     id 'groovy-gradle-plugin'
+    id 'org.jetbrains.intellij' version '1.14.1'
```
Comment (yoomlam) [inline gradle-plugins/build.gradle:8]: @greene858 Should this line be: `id 'org.jetbrains.intellij-plugin' version '1.14.1'`?
RE: [Slack](https://dsva.slack.com/archives/C04QLHM9LR0/p1688053889336159)

File: gradle-plugins/build.gradle [hunk 2]
```diff
@@ -14,19 +19,17 @@ repositories {
 }
 
 dependencies {
-    implementation  group: 'gradle.plugin.org.jetbrains.intellij.plugins', name: 'gradle-intellij-plugin', version: '0.4.26'
```
Comment (yoomlam) [inline gradle-plugins/build.gradle:17]: or maybe just update the version on this line: `version: '1.14.1'`

Comment (greene858) [reply gradle-plugins/build.gradle:17]: When version was specified in the _dependencies_ block gradle was pulling from https://plugins.gradle.org/m2/gradle/plugin/org/jetbrains/intellij/plugins/gradle-intellij-plugin/ which only went up to version 0.4.26 and we need 1.14.1. When IntelliJ was specified in the _plugin_ block it was able to pull the needed version.

**Pull request merged**